internal/pkg/externalplugins/lgtm/lgtm.go

```diff
@@ -238,17 +238,17 @@ func handle(wantLGTM bool, config *externalplugins.Configuration, rc reviewCtx,
 		reviewers.Insert(reviewer)
 	}
 
-	// Not reviewers but want add LGTM.
+	// Not reviewers but want to add LGTM.
 	if !reviewers.Has(author) && wantLGTM {
-		resp := "adding LGTM is restricted to reviewers in [list](" + url + ")."
-		log.Infof("Reply to /lgtm request with comment: \"%s\"", resp)
+		resp := "`/lgtm` is only allowed for the reviewers in [list](" + url + ")."
+		log.Infof("Reply /lgtm request in comment: \"%s\"", resp)
 		return gc.CreateComment(org, repoName, number, externalplugins.FormatResponseRaw(body, htmlURL, author, resp))
 	}
 
-	// Not author or reviewers but want remove LGTM.
```

Comment:
I think it also checked if commenter was the author.

Reply:
I've removed this feature.